org.envirocar.app/res/values/strings_activity_login.xml

```diff
@@ -22,16 +22,16 @@
     <string name="action_sign_in_register">Register</string>
     <string name="action_sign_in_short">Sign in</string>
     <string name="error_invalid_email">This email address is invalid.</string>
-    <string name="error_invalid_password">This password is too short.</string>
+    <string name="error_invalid_password">This password is too short.Password must be of atleast 6 characters.</string>
     <string name="error_incorrect_password">This password is incorrect.</string>
     <string name="error_invalid_credentials">Username or password is incorrect</string>
     <string name="error_field_required">This field is required.</string>
     <string name="error_not_connected_to_network">Not connected to network</string>
     <string name="register_progress_signing_in">Registering&#8230;</string>
     <string name="error_passwords_not_matching">The passwords do not match.</string>
-    <string name="error_invalid_username">This username is too short.</string>
+    <string name="error_invalid_username">This username is too short.Username must be  of atleast 6 characters.</string>
```

Comment:
Yes. This is a good enhancement. Kindly remove the extra space between letters and add space after fullstop.